packages/pieces/bettermode/src/lib/actions/assign-badge.ts

```diff
@@ -0,0 +1,34 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { assignBadgeToMember } from "../api";
+import { buildBadgesDropdown } from "../props";
+import { bettermodeAuth, BettermodeAuthType } from "../auth";
+
+export const assignBadgeAction = createAction({
+	name        : 'assignBadge',
```

Comment:
Action or trigger names should be snake_case, so assign_badge instead (do this for all actions pls)